### PR TITLE
[SMALLFIX] Use correct punctuations in docs/cn/index.md

### PR DESCRIPTION
**Pull request by konnase**

### CODE REVIEW

File: docs/cn/index.md [hunk 1]
```diff
@@ -58,8 +58,7 @@ InputStream和OutputStream的接口和对内存映射I/O的高效支持。我们
 不同的底层存储系统。目前我们支持Microsoft Azure Blob Store，Amazon S3，Google Cloud Storage，OpenStack Swift，GlusterFS，
 HDFS，MaprFS，Ceph，NFS，Alibaba OSS，Minio以及单节点本地文件系统，后续也会支持很多其它的文件系统。
 
-* **[Alluxio存储](Alluxio-Storage.html)** Alluxio可以管理内存和本地存储如SSD
-和HDD,以加速数据访问。如果需要更细粒度的控制，分层存储功能可以用于自动管理不同层之间的数据，保证热数据在更快的存储层上。
+* **[Alluxio存储](Alluxio-Storage.html)** Alluxio 可以管理内存和本地存储如 SSD 和 HDD，以加速数据访问。如果需要更细粒度的控制，分层存储功能可以用于自动管理不同层之间的数据，保证热数据在更快的存储层上。
```
Comment (LuQQiu) [inline docs/cn/index.md]: In our convention, we do not add space between English words and Chinese words. Thanks for your contribution!

Comment (konnase) [reply docs/cn/index.md]: I pushed a new commit, please check the correctness

**Comment by LuQQiu** [comment]
@konnase Please fix the merge conflicts.
you can use `git pull upstream master` in your local branch, fix the conflicts, and then use the same git add, git commit, git push origin commands as you did before. Thanks!

**Comment by konnase** [comment]
thanks for the quick reply! It seems that the file 'docs/cn/index.md' has been deleted, how can i make contribution to this file?

**Comment by konnase** [comment]
I found the error. This file was renamed to "doc/cn/Overview.md". Now the conflict has been resolved, please check. Thank you so much!

**Pull request merged**